components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -163,6 +163,13 @@ class BrowserToolbar @JvmOverloads constructor(
         get() = displayToolbar.onUrlClicked
         set(value) { displayToolbar.onUrlClicked = value }
 
+    /**
+     * Sets a lambda that will be invoked whenever the URL in display mode was long clicked.
+     */
+    var onUrlLongClicked: (() -> Unit)?
+        get() = displayToolbar.onUrlLongClicked
+        set(value) { displayToolbar.onUrlLongClicked = value }
```

Comment:
I wonder whether this needs to be a field or could just be a setter like other listeners in here? It would avoid some of the indirection and you could added the OnLongClickListener on the view here directly.

Reply:
I originally made it a field to be consistent with the click listener, I'll rip it out :)